pkgs/applications/audio/cozy/default.nix

```diff
@@ -57,7 +48,6 @@ python3Packages.buildPythonApplication rec {
 
   propagatedBuildInputs = with python3Packages; [
     apsw
-    cairo
     dbus-python
```

Comment:
All are of these still necessary? Either the [upstream list](https://github.com/geigi/cozy/blob/027efb7622ef38795d3cc006fb0704e3a8f86ba5/meson.build#L41) is outdated or this one is.